lib/tasks/uwm.rake

```diff
@@ -79,6 +79,25 @@ namespace :uwm do
     end
   end
 
+  namespace :opendataharvest do
+    desc "Set up Python venv environment for opendataharvest"
+    task :setup_python_env do
+      sh "lib/opendataharvest/setup_python_env.sh"
+    end
+
+    desc "Run the DCAT_Harvester.py Python script"
+    task :harvest_dcat do
+      sh "lib/opendataharvest/venv/bin/python3 lib/opendataharvest/opendataharvest/DCAT_Harvester.py"
+    end
+
+    desc "Run the conversion scripts on GBL 1.0 metadata institutions"
+    task :gbl1_to_aardvark do
```

Comment:
And this one runs gbl_to_aardvark.py which is a subprocess script that runs the convert script over the defined directories. This will make it easier to update as some of those institutions move over to aardvark.